### PR TITLE
Task 4

### PR DESCRIPTION
**Pull request by AnnaKrikun**

### CODE REVIEW

File: src/main/webapp/WEB-INF/pages/cart.jsp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    <form method="post" action="${pageContext.servletContext.contextPath}/cart">
+        <table>
+            <tr>
+                <td>Image</td>
+                <td>Description</td>
+                <td>Price</td>
+                <td>Quantity</td>
+                <td>Action</td>
+            </tr>
+            <c:forEach var="item" items="${cart.cartItems}" varStatus="status">
```
Comment (SergeiStarikov) [inline src/main/webapp/WEB-INF/pages/cart.jsp]: If the cart is empty we will get empty table with headers - consider checking for epmty table before rendering the table.

File: src/main/webapp/WEB-INF/pages/cart.jsp [hunk 2]
```diff
@@ -15,41 +15,50 @@
...
-                </td>
-            </tr>
-            </c:forEach>
-        </table>
-        <br>
-        <button>Update</button>
-    </form>
+    <c:choose>
+        <c:when test="${not empty cart.cartItems}">
+        <form method="post" action="${pageContext.servletContext.contextPath}/cart">
```
Comment (SergeiStarikov) [inline src/main/webapp/WEB-INF/pages/cart.jsp:20]: here and in other places - code structure:
```
<c:choose>
    <c:when test="${not empty cart.cartItems}">
        <form>
```

File: src/test/java/com/es/phoneshop/web/ProductListPageServletTest.java [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package com.es.phoneshop.web;
```
Comment (SergeiStarikov) [inline src/test/java/com/es/phoneshop/web/ProductListPageServletTest.java:1]: Please add more tests for boundary cases.
